lib/lookup.json

```diff
@@ -443,5 +443,9 @@
     "prefix": "v",
     "flaky": "ppc",
     "tags": "native"
+  },
+  "blake2b-wasm": {
+    "maintainers": ["mafintosh"],
+    "prefix": "v"
```

Comment:
Looking at the CI, it looks like this fails on Node <8, so could you add

```json
"skip": "<8"
```

(I think that syntax works).

Reply:
done!